### PR TITLE
feat: Add options to control event dates

### LINKED ISSUE #42: Add options for start and end date
Right now the events are spread out over a fairly significant number of years. We should make this optional so we can test data that is more realistically compressed.

### PR DESCRIPTION
**Pull request by bmtcril**
Lets us cluster events with more granularity to better test performance.

Closes #42

### CODE REVIEW

**Review by Ian2012 — APPROVED** [review]
LGTM

File: xapi_db_load/generate_load.py [hunk 1]
```diff
@@ -90,12 +90,14 @@ class RandomCourse:
     start_date = None
     end_date = None
 
-    def __init__(self, org):
+    def __init__(self, org, min_start_date, max_end_date):
```
Comment (pomegranited) [inline xapi_db_load/generate_load.py:93]: nit: why are these called `min_start_date` and `max_end_date` when everywhere else, they're called `start_date` and `end_date`?

File: xapi_db_load/generate_load.py [hunk 2]
```diff
@@ -90,12 +90,14 @@ class RandomCourse:
...
         self.course_uuid = str(uuid.uuid4())
         self.org = org
         self.course_id = f"course-v1:{org}+DemoX+{self.course_uuid}"
         self.course_url = f"http://localhost:18000/course/{self.course_id}"
 
+        self.start_date = min_start_date
+        self.end_date = max_end_date
```
Comment (pomegranited) [inline xapi_db_load/generate_load.py:100]: nit:
```suggestion
    def __init__(self, org, start_date, end_date):
        self.course_uuid = str(uuid.uuid4())
        self.org = org
        self.course_id = f"course-v1:{org}+DemoX+{self.course_uuid}"
        self.course_url = f"http://localhost:18000/course/{self.course_id}"

        self.start_date = start_date
        self.end_date = end_date
```

**Review by pomegranited — APPROVED** [review]
:+1: Works well, thanks @bmtcril !

- [x] I tested this by installing this branch with the [clickhouse-demo-xapi-data.sh](https://github.com/openedx/tutor-contrib-aspects/blob/main/tutoraspects/templates/aspects/apps/aspects/scripts/clickhouse-demo-xapi-data.sh) script and running the command with various options.
- [x] I read through the code
- [x] <strike>I checked for accessibility issues</strike> N/A
- [x] Includes documentation

**Pull request merged**